zsh_startify/__main__.py

```diff
@@ -257,8 +257,10 @@ def main():
     splash_flag = not getenv("ZSH_STARTIFY_NO_SPLASH", "")
     if splash_flag:
         splash()
-
-    server = get_tmux_server()
+    try:
+        server = get_tmux_server()
+    except KeyboardInterrupt:
+        exit("Program closed by user")
```

Comment:
`sys` should be imported at the top and `sys.exit` should be used here. Also, I don't think any output is necessary. This feature is just intended to get rid of the error traceback.

```suggestion
        sys.exit()
```

https://stackoverflow.com/a/6501134